corehq/apps/case_search/filter_dsl.py

```diff
@@ -127,15 +128,8 @@ def _parent_property_lookup(node):
         """given a node of the form `parent/foo = 'thing'`, return all case_ids where `foo = thing`
         """
         es_filter = _comparison_raw(node.left.right, node.op, node.right, node)
-        es_query = CaseSearchES().domain(context.domain).filter(es_filter)
-        if es_query.count() > MAX_RELATED_CASES:
-            new_query = '{} {} "{}"'.format(serialize(node.left.right), node.op, node.right)
-            raise TooManyRelatedCasesError(
-                _("The related case lookup you are trying to perform would return too many cases"),
-                new_query
-            )
-
-        return es_query.scroll_ids()
+        new_query = '{} {} "{}"'.format(serialize(node.left.right), node.op, node.right)
+        return _do_parent_lookup(context.domain, es_filter, new_query)
```

Comment:
nit: instead of `new_query`, maybe `query_repr` or something to indicate that it's not actually the query being run, just how it's presented to the user.

Reply:
b2e81609cbc